src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,80 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  todos: Todo[];
+  select?: number;
```

Comment:
The `select` prop is optional, but the component logic assumes it will always have a valid value. Consider handling the case where `select` is undefined to avoid potential issues.